apps/playground-web/src/app/local/page.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import { THIRDWEB_CLIENT } from "@/lib/client";
+import { Suspense } from "react";
+import { sepolia } from "thirdweb/chains";
+import { Account, Blobbie, ThirdwebProvider } from "thirdweb/react";
+import { shortenAddress } from "thirdweb/utils";
+
+export default function Page() {
+  return (
+    <ThirdwebProvider>
+      <div className="flex h-full w-full flex-col items-center gap-3">
+        <div>
+          <div>Let&apos;s rebuild a basic ConnectButton-details component</div>
+          <BasicConnectedButton />
+        </div>
+      </div>
+    </ThirdwebProvider>
+  );
+}
+
+const BasicConnectedButton = () => {
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/VtULqGKSp33n0VHOzNJK/c8f94fdc-cb6e-422c-a234-585ddf6fb1b0.png)

Reply:
just for test, remove later